From eeec7e0b09c8770763596942e646ab69a5436de7 Mon Sep 17 00:00:00 2001
From: Ori Livneh <ori@wikimedia.org>
Date: Wed, 30 Jul 2014 17:00:13 -0700
Subject: [PATCH] Expose extension API version in output of 'hhvm --version'

It's useful for packaging and configuration management scripts that want to
determine where dynamic extension files should be placed.
---
 hphp/runtime/base/program-functions.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hphp/runtime/base/program-functions.cpp b/hphp/runtime/base/program-functions.cpp
index 1a8673f..2520090 100644
--- a/hphp/runtime/base/program-functions.cpp
+++ b/hphp/runtime/base/program-functions.cpp
@@ -1188,6 +1188,7 @@ static int execute_program_impl(int argc, char** argv) {
     cout << " (" << (debug ? "dbg" : "rel") << ")\n";
     cout << "Compiler: " << kCompilerId << "\n";
     cout << "Repo schema: " << kRepoSchemaId << "\n";
+    cout << "Extension API: " << std::to_string(HHVM_API_VERSION) << "\n";
     return 0;
   }
   if (vm.count("compiler-id")) {
-- 
2.0.3

